src/xcode/ENA/ENA/Source/Services/Risk/__tests__/RiskTests.swift

```diff
@@ -23,36 +23,36 @@ import XCTest
 
 final class RiskTests: XCTestCase {
 	func testGetNumberOfDaysActiveTracing_LessThanOneDay() {
-		let details = mockDetails(hours: 23)
+		let details = mockDetails(activeTracing: .init(interval: .init(hours: 11), maximumNumberOfDays: 14))
```

Comment:
it might make sense to make maximumNumberOfDays a default parameter with a value of 14 (do we ever need other values here at all?)

Reply:
:white_check_mark: